ref/Microsoft.Build/netstandard/Microsoft.Build.cs

```diff
@@ -1422,7 +1436,7 @@ public sealed partial class ProjectGraph
         public System.Collections.Generic.IReadOnlyCollection<Microsoft.Build.Graph.ProjectGraphNode> EntryPointNodes { get { throw null; } }
         public System.Collections.Generic.IReadOnlyCollection<Microsoft.Build.Graph.ProjectGraphNode> GraphRoots { get { throw null; } }
         public System.Collections.Generic.IReadOnlyCollection<Microsoft.Build.Graph.ProjectGraphNode> ProjectNodes { get { throw null; } }
-        public System.Collections.Generic.IReadOnlyDictionary<Microsoft.Build.Graph.ProjectGraphNode, System.Collections.Immutable.ImmutableList<string>> GetTargetLists(string[] entryProjectTargets) { throw null; }
+        public System.Collections.Generic.IReadOnlyDictionary<Microsoft.Build.Graph.ProjectGraphNode, System.Collections.Immutable.ImmutableList<string>> GetTargetLists(System.Collections.Generic.ICollection<string> entryProjectTargets) { throw null; }
```

Comment:
Now that this is called from BuildManager instead of XMake, I can actually just make `GetTargetLists` internal (I think).

Any opinions on whether it should be internal (if possible)?

Reply:
Offline discussion: Users that want to distribute msbuild themselves via the new graph and cache serialization, will have to know what targets to call each graph node with in order to get those target results in the cache.